pkg/operator/starter.go

```diff
@@ -41,6 +42,18 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
+type encryptionProvider []schema.GroupResource
+
+var _ controllers.Provider = encryptionProvider{}
+
+func (p encryptionProvider) EncryptedGRs() []schema.GroupResource {
```

Comment:
@p0lyn0mial I missed this in library-go, but I think we should try to keep interfaces more verbose rather than cryptic, iow. this should've been `EncryptedGroupResources` just to follow the API patterns we have everywhere. It's ok to name local variables this way but interfaces not necessarily, if in doubt look at k8s always :wink:

Reply:
yes, thx :)